src/components/Book.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './Book.css';
+
+const Book = (props) => {
+  const { author, title, category } = props;
+  return (
+    <li className="book">
+      <div className="book-content">
+        <p>{category}</p>
+        <h3>{title}</h3>
+        <h4>{author}</h4>
+        <div className="actions">
+          <p>Comments</p>
+          <p>Remove</p>
```

Comment:
- [ ] Let's configure this component properly. There should be a remove button in this book component. You can do that by changing this to a remove button, instead of a paragraph.

Reply:
Thanks again!